pkgs/development/libraries/qt-6/modules/qtdeclarative.nix

```diff
@@ -4,24 +4,27 @@
 , qtshadertools
 , openssl
 , stdenv
-, python3
 , lib
 , pkgsBuildBuild
 }:
 
 qtModule {
   pname = "qtdeclarative";
-  strictDeps = !stdenv.isDarwin; # fails to detect python3 otherwise
```

Comment:
Any idea why?

Reply:
No idea, someone with an actual Darwin machine may want to investigate.

Reply:
https://github.com/nixOS/nixpkgs/commit/8c9c8ade2f88a85ccdd4858cc802d7b7d6c48fe0 looks suspicious.